posts/aws-and-image-builder/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Customize AWS cloud images with Image Builder and Packit
+date: 2024-01-22
+authors: mmassari
+tags:
+  - image-builder
+---
+
+Have you ever wanted to bring your pull request changes in a cloud image easily?
+How easy is it? Just comment on your pull request with `/packit vm-image-build`.
+
+With the above command, Packit automates all the manual steps needed to create a
+RPM package with your pull request changes and asks the Image Builder to install it
+inside a brand new cloud image.
+
```

Comment:
I would probably add some sentence here to smoothly connect the next sections, something like `Let's have a look at the prerequisites for this.`